docs/en/user_guides/visualization.md

```diff
@@ -76,4 +76,59 @@ we can also run the following command to view them in TensorBoard:
 tensorboard --logdir work_dirs/test_visual/20220810_115248/vis_data
 ```
 
-If you would like to know more visualization usage, you can refer to [visualization tutorial](https://mmengine.readthedocs.io/en/latest/advanced_tutorials/visualization.html) in mmengie.
+We could use `SegLocalVisualizer` to visualize results with their corresponding predictions or ground truth. For example, please follow commands below in your MMSegmentation main directory:
+
+```python
+import mmcv
+import os.path as osp
+import torch
+from mmengine.structures import PixelData
+
+from mmseg.structures import SegDataSample
+from mmseg.visualization import SegLocalVisualizer
+
+out_file = 'out_file_cityscapes'
+save_dir = './work_dirs'
+
+image = mmcv.imread(
+    osp.join(
+        osp.dirname(__file__),
+        './tests/data/pseudo_cityscapes_dataset/leftImg8bit/val/frankfurt/frankfurt_000000_000294_leftImg8bit.png'  # noqa
+    ),
+    'color')
+sem_seg = mmcv.imread(
+    osp.join(
+        osp.dirname(__file__),
+        './tests/data/pseudo_cityscapes_dataset/gtFine/val/frankfurt/frankfurt_000000_000294_gtFine_labelTrainIds.png'  # noqa
+    ),
+    'unchanged')
+sem_seg = torch.unsqueeze(torch.from_numpy(sem_seg), 0)
```

Comment:
It might not necessary to extend the dimension, as it allows input 2d data.
or if want to keep this line, I suggest add some comments, as it might make users confused about the dimension extension

https://github.com/open-mmlab/mmengine/blob/6b1b8a375163ea1b8712cf886204a1d08d17cee1/mmengine/structures/pixel_data.py#L79-L81